src/policies/user.js

```diff
@@ -0,0 +1,16 @@
+import _ from 'lodash';
+import SCOPES from '../middleware/scopeConstants';
+
+export default class Users {
+  constructor(user) {
+    this.user = user;
+  }
+
+  canViewUsersInRegion(region) {
+    const permissions = _.find(this.user.permissions,
```

Comment:
Isn't `this.user.permissions` an Array? Could we use `this.user.permissions.find()` here?